versioned/persist/serialize/src/test/java/org/projectnessie/versioned/persist/serialize/TestSerialization.java

```diff
@@ -360,6 +388,41 @@ public void stableOrderOfRepoDescProps() {
     assertThat(repoProps.getPropertiesList()).containsExactlyElementsOf(expected);
   }
 
+  @Test
+  public void attachmentContentSerialization() throws Exception {
+    for (int i = 0; i < 500; i++) {
```

Comment:
can you provide an explanation why `500` was chosen?

Reply:
I don't have a particular one - just to have some more randomness here.
500 is a bit too much - updated to 50

Reply:
If we need randomness, maybe we should consider property-based testing?